Sources/Sentry/Public/SentryTransaction.h

```diff
@@ -27,12 +27,12 @@ SENTRY_NO_INIT
  * Longer description of the span's operation, which uniquely identifies the span but is
  * consistent across instances of the span.
  */
-@property (nonatomic, copy) NSString *_Nullable spanDescription;
+@property (nullable, nonatomic, copy) NSString *spanDescription;
 
 /**
  * Short code identifying the type of operation the transaction is measuring.
  */
-@property (nonatomic, copy) NSString *_Nullable operation;
+@property (nullable, nonatomic, copy) NSString *operation;
```

Comment:
H: `name` and `operation` are always non nullale, `description` is nullable

Reply:
Right. This should be fixed in Java.